docs/developer-docs/latest/development/plugins-development.md

```diff
@@ -6,23 +6,28 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/development/plugins-d
 
 # Plugins development
 
-Strapi allows you to create local plugins that work exactly like the external ones installed with npm or through the Marketplace.
+Strapi allows the development of local plugins that work exactly like the external plugins available from the [Marketplace](). 
 
 :::strapi Extending plugins
 If you would rather extend an existing plugin than create a new one, see the [Plugins extension](/developer-docs/latest/development/plugins-extension.md) documentation.
 :::
 
-## Creating a plugin
+## Create a plugin
 
-To create a plugin, use Strapi CLI tools:
+Strapi provides a [command line interface (CLI)](/developer-docs/latest/developer-resources/cli/CLI.md) for creating plugins. To create a plugin:
 
-1. (_Optional_) If you don't already have an existing project, create a new development project with `strapi new myDevelopmentProject`.
-2. Start the project with `cd myDevelopmentProject && strapi develop`.
-3. In a new terminal window, run `cd /path/to/myDevelopmentProject && strapi generate` to launch the interactive `strapi generate` CLI.
-4. Choose "plugin" from the list, press Enter, and give the plugin a name.
-5. Enable the plugin by adding it to the [plugins configurations](/developer-docs/latest/setup-deployment-guides/configurations/optional/plugins.md) file:
-    ```js
-    // path: /path/to/myDevelopmentProject/config/plugins.js
+1. Navigate to the root of a Strapi project.
+2. Run `yarn strapi generate` or `npm run strapi generate` in a terminal window to start the interactive `Strapi generate` CLI.
```

Comment:
```suggestion
2. Run `yarn strapi generate` or `npm run strapi generate` in a terminal window to start the interactive CLI.
```

I suggested removing Strapi generate entirely since something did not feel right to me (backticks should be used for commands or file names, but then `strapi` should be lower-case — some terminal instances are case-sensitive).